src/main/java/org/prebid/server/auction/ExchangeService.java

```diff
@@ -1044,6 +1054,16 @@ private Bid toBid(BidderBid bidderBid, String bidder, TargetingKeywordsCreator k
             CacheAsset vastXml = videoCacheId != null
                     ? CacheAsset.of(cacheService.getCachedAssetURL(videoCacheId), videoCacheId) : null;
 
+            if (vastXml != null) {
```

Comment:
Proposal: to be more consistent we can use `cacheId` and `videoCacheId` variables:
```
if ((cacheId != null && !cacheInfo.returnCreativeBids) || (videoCacheId != null && !cacheInfo.returnCreativeVideoBids)) {
  bid.setAdm(null);
}
```
What do you think?

Reply:
Ok agree, let's do as you decided but we need to add check condition for `cacheId`:
```
if (videoCacheId != null) {
    if (!cacheInfo.returnCreativeVideoBids) {
        bid.setAdm(null);
    }
} else if (cacheId != null) {
    if (!cacheInfo.returnCreativeBids) {
        bid.setAdm(null);
    }
}
```